examples/standard/cluster-role.yaml

```diff
@@ -92,3 +92,16 @@ rules:
   verbs:
   - list
   - watch
+- apiGroups:
+  - admissionregistration.k8s.io
+  resources:
+  - mutatingwebhookconfigurations
+  - validatingwebhookconfigurations
+  verbs:
+  - list
+  - watch
+
```

Comment:
These empty lines are extraneous. We should have only one empty line for the EOF.

Reply:
I believe there is only one empty line atm

Reply:
I see more in my diff. L104-107.  Is that not the case for you?

Reply:
aah yes. I was looking at the other cluster-role file 🤦‍♀️